### ISSUE TITLE
Running the contributed command:'extension.arrowFunctionConverter' failed.

### ISSUE THREAD

**Issue by yegao** [DaRoee/arrow-function-converter#1]
when I use alt+shift+K,the vscode warn 'Running the contributed command:'extension.arrowFunctionConverter' failed.'

**Comment by RoeeDr** [comment]
Hey,
Which version of vscode are you using? Can you add extension host logs?

**Comment by yegao** [comment]
> Hey,
> Which version of vscode are you using? Can you add extension host logs?

Hey,the version is 1.29.1 (1.29.1)

**Comment by danielres** [comment]
I encounter the same problem

```
[2018-12-25 14:35:35.823] [exthost] [info] ExtensionService#_doActivateExtension vscode.configuration-editing {"startup":false,"activationEvent":"onLanguage:jsonc"}
[2018-12-25 14:35:35.823] [exthost] [info] ExtensionService#loadCommonJSModule /usr/share/code/resources/app/extensions/configuration-editing/dist/extension
[2018-12-25 14:35:35.830] [exthost] [info] ExtensionService#_doActivateExtension vscode.json-language-features {"startup":false,"activationEvent":"onLanguage:jsonc"}
[2018-12-25 14:35:35.830] [exthost] [info] ExtensionService#loadCommonJSModule /usr/share/code/resources/app/extensions/json-language-features/client/dist/jsonMain
[2018-12-25 14:35:56.309] [exthost] [info] ExtensionService#_doActivateExtension Roee.arrow-function-converter {"startup":false,"activationEvent":"onCommand:extension.arrowFunctionConverter"}
[2018-12-25 14:35:56.309] [exthost] [info] ExtensionService#loadCommonJSModule /home/daniel/.vscode/extensions/roee.arrow-function-converter-0.0.6/out/extension
[2018-12-25 14:35:56.321] [exthost] [error] TypeError: Cannot read property 'split' of undefined
	at vscode.commands.registerCommand (/home/daniel/.vscode/extensions/roee.arrow-function-converter-0.0.6/out/extension.js:24:37)
	at e._executeContributedCommand (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:501:455)
	at e.$executeContributedCommand (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:502:6)
	at t._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:658:837)
	at t._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:658:509)
	at t._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:657:126)
	at t._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:656:59)
	at /usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:653:895
	at /usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:130:793
	at e.fire (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:133:218)
	at a (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:186:147)
	at Socket.n._socketDataListener (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:186:368)
	at emitOne (events.js:116:13)
	at Socket.emit (events.js:211:7)
	at addChunk (_stream_readable.js:263:12)
	at readableAddChunk (_stream_readable.js:250:11)
	at Socket.Readable.push (_stream_readable.js:208:10)
	at Pipe.onread (net.js:594:20) extension.arrowFunctionConverter
```

VSCode info:

![image](https://user-images.githubusercontent.com/168687/50423184-8dbfab80-0852-11e9-860d-9a3bd7cfa0e3.png)

And here is the code I've been trying to use the extension with:

```
function ticked() {
  var u = select('#Force svg')
    .selectAll('circle')
    .data(nodes)

  u.enter()
    .append('circle')
    .attr('r', 2)
    .style('fill', 'white')
    .merge(u)
    .attr('cx', function(d) {
      return d.x
    })
    .attr('cy', function(d) {
      return d.y
    })

  u.exit().remove()
}
```

Specifically, I tried to convert:

```
.attr('cx', function(d) {
      return d.x
    })
```

I positioned the cursor inside, before, after the block in different places, but no success.

Thanks

**Comment by RoeeDr** [comment]
@danielres What are you trying to do here?
The code you wrote there is not a 1 liner, and currently the extension is only supporting one liner to multiline (will add the opposite in the future) :)
@yegao What about you? Is the issue solved in 0.1.0?
